uktrade/data-hub-frontend
# Add full export history page (no pagination)

## Description of change

This adds the full export history page. In this first iteration there is no pagination or axios calls. It simply fetches all the data and displays it in one go. Pagination will be in a later PR. 

This is also a hidden URL at the moment. I will create a seperate PR to add the link to this page to the export page. 

All additions and deletions are currently shown as individual events. There is a ticket for the backend to group these together to reflect when countries were added and deleted in a single user event. I will further iterate this front end feature to incorporate this when it is ready. 

## Test instructions

You should see the export countries history using the collectionList component when going to /companies/:companyId/exports/history

There is only history from when we merged the history PR last week - so you may have to add and delete some export countries to see the feature in action. 

## Screenshots

![FireShot Capture 034 - Full export history - Exports - One List Subsidiary Ltd - Companies -_ - localhost](https://user-images.githubusercontent.com/42253716/74151016-1f2ee000-4c03-11ea-96df-dc8ed0687db8.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
